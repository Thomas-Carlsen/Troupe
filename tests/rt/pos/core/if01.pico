let val h = 1
    fun foo x = x + 1

in  let val _ =  if h then foo 21 else 0
     in printWithLabels 42
    end

end
