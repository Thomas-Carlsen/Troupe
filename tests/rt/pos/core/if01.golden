2019-03-07T12:08:05.415Z [RTM] [32minfo[39m: Skipping network creation. Observe that all external IO operations will yield a runtime error.
42@{}%{}
>>> Main thread finished with value: ()@{}%{}
